### PR TITLE
Remove hard coded ballerina version in java code

### PR DESCRIPTION
**Pull request by MadhukaHarith92**

### CODE REVIEW

File: math-native/src/main/java/org/ballerinalang/stdlib/math/nativeimpl/Constant.java [hunk 1]
```diff
@@ -35,9 +35,15 @@
 
     public static final String OVERFLOW_ERROR_MSG = "Overflow occurred";
 
-    private static final String PACKAGE_NAME = "math";
-
+    /**
+     * Math standard library version.
+     * @deprecated Use MathUtils.getMathPackage().getVersion().
+     */
     public static final String MATH_PACKAGE_VERSION =  "1.0.1";
```
Comment (ThisaruGuruge) [inline math-native/src/main/java/org/ballerinalang/stdlib/math/nativeimpl/Constant.java:42]: We can remove this completely right? Are we using this anywhere else other than the math module?

File: math-native/src/main/java/org/ballerinalang/stdlib/math/nativeimpl/Constant.java [hunk 2]
```diff
@@ -35,9 +35,15 @@
...
 
-    static final Module MATH_PACKAGE_ID = new Module(BALLERINA_BUILTIN_PKG_PREFIX, PACKAGE_NAME, MATH_PACKAGE_VERSION);
+    /**
+     * Math standard library package ID.
+     * @deprecated Use MathUtils.getMathPackage().
+     */
+    static final Module MATH_PACKAGE_ID = new Module(BALLERINA_BUILTIN_PKG_PREFIX, "math", MATH_PACKAGE_VERSION);
```
Comment (ThisaruGuruge) [inline math-native/src/main/java/org/ballerinalang/stdlib/math/nativeimpl/Constant.java:48]: We can remove this too, right?

File: math-native/src/main/java/org/ballerinalang/stdlib/math/utils/MathUtils.java [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+import org.ballerinalang.stdlib.math.nativeimpl.ModuleUtils;
+
+public class MathUtils {
+
+    /**
+     * Gets ballerina math package.
+     *
+     * @return math package.
+     */
+    public static Module getMathPackage() {
```
Comment (ThisaruGuruge) [inline math-native/src/main/java/org/ballerinalang/stdlib/math/utils/MathUtils.java:32]: Why do we need two separate methods to do the same thing?

File: math-native/src/main/java/org/ballerinalang/stdlib/math/nativeimpl/Constant.java (hunk 1 above)
Comment (MadhukaHarith92) [reply math-native/src/main/java/org/ballerinalang/stdlib/math/nativeimpl/Constant.java]: Removed in 3e935cb

File: math-native/src/main/java/org/ballerinalang/stdlib/math/nativeimpl/Constant.java (hunk 2 above)
Comment (MadhukaHarith92) [reply math-native/src/main/java/org/ballerinalang/stdlib/math/nativeimpl/Constant.java]: Removed in 3e935cb

File: math-native/src/main/java/org/ballerinalang/stdlib/math/nativeimpl/Constant.java [hunk 4]
```diff
@@ -34,10 +34,4 @@
     public static final String DIVIDE_BY_ZERO_ERROR_MSG = "Division by zero occurred";
 
     public static final String OVERFLOW_ERROR_MSG = "Overflow occurred";
-
-    private static final String PACKAGE_NAME = "math";
-
-    public static final String MATH_PACKAGE_VERSION =  "1.0.1";
-
-    static final Module MATH_PACKAGE_ID = new Module(BALLERINA_BUILTIN_PKG_PREFIX, PACKAGE_NAME, MATH_PACKAGE_VERSION);
```
Comment (ThisaruGuruge) [inline math-native/src/main/java/org/ballerinalang/stdlib/math/nativeimpl/Constant.java:42]: The `BALLERINA_BUILTIN_PKG_PREFIX` is now an unused import, right?

This gets me to think, is spotbugs and the checkStyle enabled for this repo? If not, shall we enable them?

File: math-native/src/main/java/org/ballerinalang/stdlib/math/utils/MathUtils.java (hunk 3 above)
Comment (MadhukaHarith92) [reply math-native/src/main/java/org/ballerinalang/stdlib/math/utils/MathUtils.java]: Removed in 3a2e918

File: math-native/src/main/java/org/ballerinalang/stdlib/math/nativeimpl/Constant.java (hunk 4 above)
Comment (MadhukaHarith92) [reply math-native/src/main/java/org/ballerinalang/stdlib/math/nativeimpl/Constant.java:42]: Enabled in 1475b4e

**Review by ThisaruGuruge — APPROVED** [review]

**Pull request merged**